kuma/core/pipeline/cleancss.py

```diff
@@ -6,5 +6,7 @@ class CleanCSSCompressor(SubProcessCompressor):
     def compress_css(self, css):
         binary = settings.PIPELINE.get('CLEANCSS_BINARY', 'cleancss')
         args = settings.PIPELINE.get('CLEANCSS_ARGUMENTS', '')
-        command = (binary, args) if args else (binary,)
+        # If the arguments ever include quoted arguments with spaces then
+        # the simple split() call here is not going to be good enough.
+        command = (binary,) + tuple(args.split())
```

Comment:
The existing code worked for a single argument but was not written to work with more than one.